app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt

```diff
@@ -61,17 +61,19 @@ class EventViewHolder(override val containerView: View) : RecyclerView.ViewHolde
 
         setFabBackground(event.favorite)
 
-        event.eventType.let {
-            if (it != null)
-                addchips(it.name)
-        }
-        event.eventTopic.let {
-            if (it != null)
-                addchips(it.name)
-        }
-        event.eventSubTopic.let {
-            if (it != null)
-                addchips(it.name)
+        if (containerView.chipTags != null) {
+            event.eventType.let {
+                if (it != null)
```

Comment:
Remove this

Reply:
If i remove this how i will be checking chipType  for favourite and similiar events cards